airflow/cli/commands/variable_command.py

```diff
@@ -76,7 +77,24 @@ def variables_import(args):
 
 def variables_export(args):
     """Exports all the variables to the file."""
-    _variable_export_helper(args.file)
+    var_dict = {}
+    with create_session() as session:
+        query = session.query(Variable).all()
+
+        data = json.JSONDecoder()
+        for variable in query:
```

Comment:
```suggestion
        for row in rows:
```
and this combined with comment - https://github.com/apache/airflow/pull/31854/files#r1228845111